dev/com.ibm.ws.security.oidc.server_fat.claimPropagation/fat/src/com/ibm/ws/security/openidconnect/server/fat/claimPropagation/CommonTests/BasicIdTokenClaimPropagationTests.java

```diff
@@ -0,0 +1,766 @@
+    }
+
+    /**
+     * The Third party OP will populate the idToken that it creates with the groupIds claim (by default)
+     * The Third party OP populate the idToken with all extra test claims
+     * The intermediate OP does not have thirdPartyIDTokenClaims configured.
+     * No third party OP claims will be included in the idToken produced by the intermediate OP.
+     * (groupIds, testProps3, testProps4 will exist in the idToken based on the registry used by the intermed OP in the test
+     * instance)
+     *
+     * @throws Exception
+     */
+    @Test
+    public void ThirdPartyIDTokenClaims_none_3rdPartyPropagates() throws Exception {
+
+        String appName = "noExtraClaims_3rdPartyPropagates";
+        WebConversation wc = new WebConversation();
+        TestSettings updatedTestSettings = testSettings.copyTestSettings();
+        updatedTestSettings.setTestURL(testSettings.getTestURL().replace("SimpleServlet", "simple/" + appName));
+        setParmValues(updatedTestSettings.getUserName());
+
+        List<validationData> expectations = vData.addSuccessStatusCodes(null);
+        expectations = setGroupIdsExpectations(expectations, thirdPartyPropagatedIdTokenFalse);
+        expectations = setTestParmExpectations(expectations, noThirdPartyParm, thirdPartyPropagatedIdTokenFalse);
+        expectations = setRealmNameExpectations(expectations);
+        expectations = setIssExpectations(expectations, appName);
+        expectations = setAudExpectations(expectations, appName);
+
+        genericRP(_testName, wc, updatedTestSettings, steps, expectations);
+    }
+
+    /**
+     * The Third party OP will populate the idToken that it creates with the groupIds claim (by default)
+     * The Third party OP populate the idToken with all extra test claims
+     * The intermediate OP does have thirdPartyIDTokenClaims configured with "groupIds".
+     * Only the third party OP groupIds claim will be included in the idToken produced by the intermediate OP.
+     * (testProps3, testProps4 will exist in the idToken based on the registry used by the intermed OP in the test instance)
+     *
+     * @throws Exception
+     */
+    @Test
+    public void ThirdPartyIDTokenClaims_groupIds_3rdPartyPropagates() throws Exception {
+
+        String appName = "propagateGroupIdsIdTokenClaims_3rdPartyPropagates";
+        WebConversation wc = new WebConversation();
+        TestSettings updatedTestSettings = testSettings.copyTestSettings();
+        updatedTestSettings.setTestURL(testSettings.getTestURL().replace("SimpleServlet", "simple/" + appName));
+        setParmValues(updatedTestSettings.getUserName());
+
+        List<validationData> expectations = vData.addSuccessStatusCodes(null);
+        expectations = setGroupIdsExpectations(expectations, thirdPartyPropagatedIdTokenTrue);
+        expectations = setTestParmExpectations(expectations, noThirdPartyParm, thirdPartyPropagatedIdTokenFalse);
```

Comment:
This is confusing again when both `thirdPartyPropagatedIdTokenTrue` and `thirdPartyPropagatedIdTokenFalse` are used. The test is about third party propagating having its claims propagated, so I'd expect `thirdPartyPropagatedIdTokenTrue` to be the only value I'd see in this test.